data/scenarios/Challenges/bridge-building.yaml

```diff
@@ -157,6 +157,7 @@ robots:
       - counter
       - dictionary
       - grabber
+      - handle
```

Comment:
@kostmo I also added a `handle` device to the `bridge-building` challenge.  I think it makes it a little more convenient but for some parts it's still way quicker to write some code to automate it, so it doesn't detract from the challenge.  But if you feel strongly it *shouldn't* be there, we can take it out!